ui/widgets/layout/src/components/styled-slots.ts

```diff
@@ -33,11 +33,9 @@ export const TopbarSlot: React.FC<ExtensionPointProps> = styled(ExtensionPoint)`
 
 export const PluginSlot: React.FC<ExtensionPointProps> = styled(BaseStyledSlot)`
   margin-top: 0.5em;
+  width: 100%;
   ${props => css`
-    @media screen and (min-width: ${props.theme.breakpoints.small.value}px) {
-      width: 30rem;
```

Comment:
why remove this value?

Reply:
The 30rem was too high for the screen size, it was getting offscreen. The plugin slot is no longer displayed at all due to that, so I figured no need for its styling either?